pkg/daemon/daemon.go

```diff
@@ -513,7 +513,37 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 
 	ptpHAEnabled := len(listHaProfiles(nodeProfile)) > 0
 
+	var clockType event.ClockType
+	profileClockType, found := (*nodeProfile).PtpSettings["clockType"]
+	if found {
+		switch profileClockType {
+		case "T-GM":
+			clockType = event.GM
+		case "T-BC":
+			clockType = event.BC
+		default:
+			clockType = event.ClockUnset
+		}
+	} else {
+		clockType = event.ClockUnset
+	}
+
+	origionalOutput := &ptp4lConf{}
+	err = origionalOutput.populatePtp4lConf(configInput)
```

Comment:
How is configInput not empty here?

Reply:
Yeah, thanks for spotting this. Also looks like populatePtp4l is parsing more than ptp4l. I'll general parsing back to the loop. And I'll call it on `nodeProfile.Ptp4lConf` here.